packages/openapi3/test/security.test.ts

```diff
@@ -246,6 +247,21 @@ describe("openapi3: security", () => {
     deepStrictEqual(res.security, [{ OpenIdConnectAuth: [] }]);
   });
 
+  it("set a unsupported auth", async () => {
+    const diagnostics = await diagnoseOpenApiFor(
+      `
+      @service
+      @useAuth({})
+      namespace MyService {}
+      `
+    );
+    expectDiagnostics(diagnostics, [
+      {
+        code: "@typespec/openapi3/unsupported-auth",
+      },
+    ]);
```

Comment:
```suggestion
    expectDiagnostics(diagnostics, {
      code: "@typespec/openapi3/unsupported-auth",
    });
```

don't need the extra array when testing a single diagnostic